ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-ref.yaml

```diff
@@ -81,6 +81,12 @@ ref:
       pool option in install-config.yaml, or when the subnets in Local Zones have been provided. EC2 Instances in Local Zones usually takes
       longer to be ready (Node Ready), so it requires longer time to wait for Node VS Machine count. This flag will extend three times
       than the regular cluster, so it would prevent the monitor to fail before starting the tests.
+  - name: TEST_SDN_LIVE_MIGRATION_OPTIONS
+    default: ""
+    documentation: |-
+      Default is blank which means no live migration from OpenShift SDN to OpenShift OVN-Kubernetes. Valid entry maybe multiple
+      key=value seperated by comma. Valid keys are 'target-cni' and 'rollback'. For 'target-cni', a valid value is 'OpenShiftSDN'
+      or 'OVNKubernetes'. For 'rollback', valid value is 'true' or 'false'.
```

Comment:
can you also add what is actually going to happen when openshift-tests is run with all of these possible KV pairs? It's not easy to figure out what is going on with this job for me.

Reply:
Added more context. Let me know what you think.